### PR TITLE
Improve description of keyboard shortcuts

### PR DESCRIPTION
**Pull request by ariasuni**
All shortcuts description starts with “to”: “to open post”, “to favourite”, etc. It’s not useful and there a lot of precedent to not use “to” in this case, even though it usually is the rule in English. Also some languages use longer words so it’s always useful to make sentences simpler.

Examples of shortcuts windows without “to”:

Twitter shortcuts window

Ubuntu Unity’s screenshot window

GNOME’s file browser screenshot windows

---

Changes:

- Remove “to” before verbs
- Use capitals
- Improve some translations
- Use curved quotations marks instead of straight ones

Final result:
Screenshot 2021-04-28 at 22-59-54 Mastodon

### CODE REVIEW

**Review by ClearlyClaire — APPROVED** [review]

File: config/locales/devise.en.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ en:
         subject: 'Mastodon: Security key authentication enabled'
         title: Security keys enabled
     omniauth_callbacks:
-      failure: Could not authenticate you from %{kind} because "%{reason}".
+      failure: Could not authenticate you from %{kind} because “%{reason}”.
```
Comment (YerinAlexey) [inline config/locales/devise.en.yml:81]: This line doesn't look related to this change, other than that - LGTM

**Review by YerinAlexey — APPROVED** [review]

File: config/locales/devise.en.yml (hunk 1 above)
Comment (ariasuni) [reply config/locales/devise.en.yml:81]: As you can see I made a commit to change the two occurrences of straight quotes in translation:

Use curved quotations marks instead of straight ones
https://github.com/tootsuite/mastodon/pull/16129/commits/6336ca771d360694f7a6725cfd286d51529bc97f

Now it’s unrelated but it’s a very small change and uncontroversial I believe, since curved quotes are also used everywhere on https://docs.joinmastodon.org/.

Comment (YerinAlexey) [reply config/locales/devise.en.yml:81]: Oh, didn't notice that this was multiple commits, it's fine then
